admin/load.php

```diff
@@ -609,18 +641,97 @@ function perflab_deactivate_plugin() {
 }
 add_action( 'admin_action_perflab_deactivate_plugin', 'perflab_deactivate_plugin' );
 
+// WordPress AJAX action to handle the button click event.
+add_action( 'wp_ajax_perflab_install_activate_standalone_plugins', 'perflab_install_activate_standalone_plugins_callback' );
+
 /**
- * Callback function hooked to admin_notices to render plugin activate/deactivate notices.
+ * Handles the standalone plugin install and activation via AJAX.
  *
  * @since n.e.x.t
- *
- * @return void
  */
-function perflab_plugin_admin_notices() {
-	if ( 'settings_page_perflab-modules' !== get_current_screen()->id ) {
-		return;
+function perflab_install_activate_standalone_plugins_callback() {
+	if ( ! wp_verify_nonce( $_REQUEST['nonce'], 'perflab-install-activate-plugins' ) ) {
+		$status['errorMessage'] = esc_html__( 'Invalid nonce: Please refresh and try again.', 'performance-lab' );
```

Comment:
Again, escaping will be handled by `wp_json_encode()`.

```suggestion
		$status['errorMessage'] = __( 'Invalid nonce: Please refresh and try again.', 'performance-lab' );
```